src/charts/bar/Bar.js

```diff
@@ -158,36 +158,40 @@ class Bar extends Component {
         shouldShowLoadingState: false,
     }
 
-    constructor(props) {
-        super(props);
+    componentDidMount() {
+        if (this.props.data !== null) {
```

Comment:
We don´t need this check anymore, right?

Reply:
You're right, removing!

Reply:
Mmm nevermind, we do need a check like this, but as you suggested below it probably should be more about `shouldShowLoadingState`.